src/SFA.DAS.AANHub.Api/Controllers/ApprenticesController.cs

```diff
@@ -63,6 +66,30 @@ public async Task<ActionResult<GetApprenticeMemberResult>> GetApprentice(long ap
             return new OkObjectResult(apprenticeMemberResult);
 
         }
-    }
-}
 
+        /// <summary>
+        /// Patch an apprentice member
+        /// </summary>
+        /// <param name="userId"></param>
+        /// <param name="apprenticeid"></param>/// 
+        [HttpPatch]
+        [Route("{apprenticeid}")]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(NoContentResult), StatusCodes.Status204NoContent)]
+        public async Task<IActionResult> PatchApprentice([FromHeader(Name = Constants.PostRequestHeaders.RequestedByUserHeader), Required] Guid? userId, [FromRoute] long apprenticeid, [FromBody] JsonPatchDocument<ApprenticeMemberPatchModel> request)
+        {
+            _logger.LogInformation("AAN Hub API: Received command to add apprentice by ApprenticeId: {apprenticeId} and UserId: {userId}", apprenticeid, userId);
```

Comment:
Message should be updated to be more reflective that you're patching and not adding

Reply:
Message text updated. Verb 'add' replaced with 'patch'